setup.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from setuptools import setup, find_packages
+
+root = Path(__file__).resolve().parent
+
+with open(root / "README.md", encoding="utf-8") as f:
+    long_description = f.read()
+
+
+def parse_reqs(name):
+    with open(root / "requirements" / name, "r") as f:
+        return f.readlines()
+
+
+reporting_reqs = parse_reqs("reporting.txt")
+dev_reqs = parse_reqs("linters.txt") + parse_reqs("unittests.txt")
+
+all_reqs = reporting_reqs + dev_reqs
+
+
+setup(
+    name="omniscripts",
+    version="0.1",
```

Comment:
let's use [semver notation](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase)
```suggestion
    version="0.1.0",
```